Extensions/xkit_patches.js

```diff
@@ -184,6 +184,24 @@ XKit.extensions.xkit_patches = new Object({
 
 	patches: {
 		"7.9.1": function() {
+
+			XKit.post_listener.observer = new MutationObserver(mutations => {
+				const criteria = XKit.page.react ? "[data-id]" : ".post_container, .post";
+				const new_posts = mutations.find(({addedNodes}) => {
+					for (let i = 0; i < addedNodes.length; i++) {
+						const $addedNode = $(addedNodes[i]);
+						if ($addedNode.is(criteria) || $addedNode.find(criteria).length) {
+							return true;
+						}
+					}
+				}) !== undefined;
```

Comment:
this is a total nitpick but for this function I think a `return false` at the end would look nicer than the `!== undefined`

although really undefined is already falsy but that's a rabbit hole into a wonderland where javascript is busy drinking tea

Reply:
this comparison is against the result of `.find`, which returns `undefined` if it finds nothing, so this is just turning the result into a boolean rather than a truthy/falsy value

Reply:
Ohhhh, I really wasn't firing on all cylinders when I was looking at this. I think https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some is nicer then?